src/simulation/m_bubbles_EE.fpp

```diff
@@ -305,7 +305,7 @@ contains
                             else
                                 rddot = f_rddot(myRho, myP, myR, myV, R0(q), &
                                                 pb_local, pbdot, alf, n_tait, B_tait, &
-                                                bub_adv_src(j, k, l), divu_in%sf(j, k, l), &
+                                                bub_adv_src(j, k, l), real(divu_in%sf(j, k, l), kind=wp), &
```

Comment:
We really shouldn't be using intrinsics within subroutine calls... Would be better to add a variable `my_divu = real(divu_in%sf(j,k,l), kind=wp)` and pass that to `s_advance_step`.